nanovdb/nanovdb/Readme.md

```diff
@@ -3,10 +3,11 @@
 # NanoVDB:
 A lightweight GPU friendly version of VDB initially targeting rendering applications.
 
-* [Build instructions for make and cmake](../../doc/nanovdb/HowToBuild.md)
-* [Frequently asked questions](../../doc/nanovdb/FAQ.md)
-* [Source tree](../../doc/nanovdb/SourceTree.md)
-* [Examples](../../doc/nanovdb/HelloWorld.md)
+* [Build instructions for make and cmake](docs/HowToBuild.md)
+* [Frequently asked questions](docs/FAQ.md)
+* [Grid cells vs grid nodes](docs/GridCells_vs_GridNodes/Main.pdf)
+* [Source tree](docs/SourceTree.md)
+* [Examples](docs/HelloWorld.md)
```

Comment:
Please revert this to the previous version:
```
* [Build instructions for make and cmake](../../doc/nanovdb/HowToBuild.md)
* [Frequently asked questions](../../doc/nanovdb/FAQ.md)
* [Source tree](../../doc/nanovdb/SourceTree.md)
* [Examples](../../doc/nanovdb/HelloWorld.md)
```
We are using the `../../doc/` directory because we are consolidating the notes from the OpenVDB and OpenVDB AX under that same directory.

Secondly, you may want to upload `Main.pdf` to this repository if you want to include that in this repository.

Reply:
reverted Readme.md as requested